### PR TITLE
Improve main readme

### PR DESCRIPTION
**Pull request by satyarohith**
Preview: https://github.com/satyarohith/command-sets/tree/docs

We have a badge now! 🎉 
command sets count

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,153 +1,108 @@
...
+- [License](#license)
+- Additional resources
+  - [Commander reference manual](https://nimbella.com/resources-commander/reference)
 
 ## Installation
 
-Make sure you've Nimbella Commander installed. If not, click [here](https://slack.com/oauth/authorize?client_id=876870346995.892105847680&scope=commands) to install it on your slack workspace.
+### Slack
+
+Click on the below to install Nimbella Commander on your Slack workspace.
```
Comment (bhageena) [inline README.md]: nit: could be reworded as `Click on the below link to add`

**Review by bhageena — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,153 +1,108 @@
...
+[Nimbella Commander](https://nimbella.com/product/commander) is the best, fastest and easiest way to build bots and commands for messaging platforms such as [Slack](https://slack.com/apps/AS833QXL0-nimbella-commander), Microsoft Teams, and Mattermost. Commander requires no new accounts to setup, no command line tools to start, and you will create your first command in seconds. The development experience can extend from your browser, to your terminal, so that you can develop, test, run and integrate bots and commands where it is most convenient for your workflow.
+
+Unique features of Nimbella Commander:
+
+1. **Command Sets:** a convenient way to bundle, share, and install commands into your messaging platform.
+2. **Access Control:** control which users can run, edit, or administer commands
+3. **Secrets Management:** securly bind API keys to commands
+4. **Audit Logs and Operator Dashboard:** monitor and administer usage.
```
Comment (rabbah) [inline README.md:15]: End all with .

File: README.md [hunk 3]
```diff
@@ -1,153 +1,108 @@
...
+  - [Installing a command set](#Installing-a-Command-Set)
+  - [Catalog of command sets](#Catalog-of-Command-Sets)
+- [Support](#support)
+- [License](#license)
+- Additional resources
+  - [Commander reference manual](https://nimbella.com/resources-commander/reference)
 
 ## Installation
 
-Make sure you've Nimbella Commander installed. If not, click [here](https://slack.com/oauth/authorize?client_id=876870346995.892105847680&scope=commands) to install it on your slack workspace.
```
Comment (rabbah) [inline README.md:11]: Add short intro Para here as I. The installation instructions for commander depend on the platform used  select the one that works for you.

File: README.md (hunk 1 above)
Comment (rabbah) [reply README.md]: Click the link below to install ...

File: README.md [hunk 4]
```diff
@@ -1,153 +1,108 @@
...
+
+### Teams
+
+Coming soon.
+
+### Mattermost
+
+Coming soon.
+
+## Command Sets
```
Comment (rabbah) [inline README.md]: Drop a subsection if it has no content? That is it need not be anchor.

File: README.md [hunk 5]
```diff
@@ -1,153 +1,108 @@
...
+```
 /nc csm_install <command-set>
 ```
 
 Replace `<command-set>` with any of the command-sets available.
 
 Example:
 
-To install AWS command-set, we have to run the below command.
+To install [AWS Command Set](aws), we have to run the below command.
```
Comment (rabbah) [inline README.md]: To install the aws command set, you have to run the command shown below.

**Comment by rabbah** [comment]
A very nice improvement.

**Pull request merged**